terraform/vault/lb.tf

```diff
@@ -17,7 +17,7 @@ resource "aws_security_group" "vault_lb_sg" {
     from_port   = 8200
     to_port     = 8200
     protocol    = "tcp"
-    cidr_blocks = ["10.49.0.0/16", "10.51.0.0/16"]
+    cidr_blocks = ["10.49.0.0/16", "10.51.0.0/16", "10.48.236.0/22", "10.48.240.0/22"]
```

Comment:
https://mana.mozilla.org/wiki/display/IT/Mozilla+Corporate+VPN#MozillaCorporateVPN-RelevantIPs

There are blocks for mdc1 stage and for the ber3 endpoint that aren't listed.
I always suggest adding stage (so when we do VPN testing, it works like prod) and 10.64.0.0/16 to cover all future endpoints.

Reply:
Should it include the "External VPN" cidr blocks?

Reply:
No, external shouldn't have the expectation of any special treatment / permissions.